ios/NewNode VPN/NewNode VPN/Colors.xcassets/gradient_off1.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "0x4D",
+          "green" : "0x91",
+          "red" : "0xFF"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "dark"
+        }
+      ],
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "1.000",
+          "green" : "1.000",
+          "red" : "1.000"
```

Comment:
These cause the app to appear all white in Dark Mode. (all of the colors seem to have this problem)

Reply:
You are right. Fixed in 921d78fc072ea322b3b01999f13130956a7d9754. I also inverted colors for the Info view in Dark Mode.